go.mod

```diff
@@ -5,29 +5,23 @@ require (
 	github.com/Azure/go-autorest v11.1.0+incompatible // indirect
 	github.com/dgrijalva/jwt-go v0.0.0-20160705203006-01aeca54ebda // indirect
 	github.com/evanphx/json-patch v4.1.0+incompatible // indirect
-	github.com/gogo/protobuf v0.0.0-20170330071051-c0656edd0d9e // indirect
```

Comment:
Since this module is deleted from `go.mod`, why it still exists under `vendor` directory?

Reply:
I have tested this locally and confirmed that this change is by design.